src/styles/blocks/section3.scss

```diff
@@ -0,0 +1,31 @@
+.section_3 {
+  display: flex;
+  height: 40px;
+  width: 166px;
+  justify-content: stretch;
```

Comment:
The 'justify-content: stretch' property has no effect on flex items in a single-line flex container (i.e., when 'flex-wrap' is 'nowrap', which is the default). If you want the flex items to fill the space, consider using 'justify-content: space-between' or 'justify-content: space-around'.